### DISCUSSION TITLE
Call for feedback about the Table of Content component

### DISCUSSION THREAD

**Discussion by gabrielwalt** [adobe/aem-core-wcm-components#1822] [Ideas]
Hi all,

Adobe is going to pick up the development of a Table of Content component in the next days.

The specs for that component have been published recently on the following wiki page:
https://github.com/adobe/aem-core-wcm-components/wiki/Table-of-Content-Component

If you have interest in such a component, please review the specs and provide us with feedback in this thread.

The implementation will happen on the issue [#1816](https://github.com/adobe/aem-core-wcm-components/issues/1816).

Best,
Gabriel

**Comment by gabrielwalt** [comment]
Did some changes to the spec:
* Removed the requirement that components like Carousel, Tabs and Accordion need to be capable of navigating to the title clicked in the ToC when these components don't yet display that title and moved that to a section "Known Limitations".
* Added the "ID" edit dialog option to specify the HTML ID on the ToC component, like it is the case for all other Core Components.
* Created the "Model JSON" section to specify the JSON output of the component, mostly just listing the configuration options.

**Comment by dartoism** [comment]
Hi,

we plan to use this component a soon as it's released. what's the progression of this development, is there any release date planned?

Thanks!

**Comment by artus** [comment]
Hi

My team is also interested in using this component. Could you please tell us when this would be available?

Kind regards

**Comment by vsarya** [comment]
@dartoism @artus The Table of Contents Component PR has been merged yesterday - https://github.com/adobe/aem-core-wcm-components/commit/ff2b76daf7e569fa58e061af96d0db7be56e58a0
It will soon be a part of the next core components release.
Meanwhile you can through the component [README](https://github.com/adobe/aem-core-wcm-components/tree/main/content/src/content/jcr_root/apps/core/wcm/components/tableofcontents/v1/tableofcontents#table-of-contents-v1) and see what all features you can expect from this component -

**Comment by sozores** [comment]
Hi!
My team is also interested in this component, are there any news about it? 
We've seen the documentation is out there, but its inclusion on the list of available components was reverted in code.

Best regards
